packages/terra-pills/src/Private/Pill.module.scss

```diff
@@ -0,0 +1,197 @@
+@import '~terra-mixins/lib/Mixins';
+
+// Themes
+@import './../clinical-lowlight-theme/Pill.module';
+@import './../orion-fusion-theme/Pill.module';
+
+:local {
```

Comment:
Can we separate the styles into different files as per the jsx component?

Reply:
Yes we can, there are more changes to existing styles soon. So will sort it out then.